insonmnia/worker/options.go

```diff
@@ -35,6 +35,10 @@ const (
 	exportKeystorePath    = "/var/lib/sonm/worker_keystore"
 )
 
+var (
+	leakedInsecureKey = common.HexToAddress("0x8125721c2413d99a33e351e1f6bb4e56b6b633fd")
```

Comment:
maybe will be better to put it into the default worker's config rather than hardcoding the address?